src/views/UIOptionsPopup/style.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const UIOptionsDialog = styled.div`
+  top: 40px;
+  right: 16px;
+  width: 38em;
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.9);
+  background-opacity: 0.5;
+  border: 2px solid silver;
```

Comment:
More uniformity points, I'd use the same grey that the UI library is using, so change `silver` to `#434343`

Reply:
>thumbup<

Reply:
Done